wetmap/src/components/newModals/shopModal/index.tsx

```diff
@@ -47,6 +51,14 @@ export default function ShopModal(props: ShopModalProps) {
   };
 
 
+  const openTripCreatorList = async () => {
+    modalContext.modalCancel();
+    modalShow(TripCreatorListModal, {
+      size: 'medium',
+    });
+  };
+
+
```

Comment:
You can use `keepPreviousModal: true` to keep shop modal "alive". In that case you don't need to manually close shop modal and then open shop modal again after user closes TripCreatorListModal. 
Please check how this property is used, for example on settings page

Reply:
Thanks for the tip. Implemented